### DISCUSSION TITLE
How to export the lint results

### DISCUSSION THREAD

**Discussion by CurlyBytes** [super-linter/super-linter#2952] [Q&A]
hi everyone, i seen some blog articles that we can extract the lint results
[here](https://adinermie.com/publishing-github-super-linter-terraform-quality-checks-to-azure-devops-pipelines/)
and [check the comment](https://blog.tyang.org/2020/06/27/use-github-super-linter-in-azure-pipelines/) it seems it is not working anymore, im looking the docs, there is no such **OUTPUT_DETAILS** nor  **OUTPUT_FORMAT** please advise

**Comment by ferrarimarco** [comment]
Can you open a feature request about this? Thanks!
